modules/server_stats.ts

```diff
@@ -46,8 +32,36 @@ const refreshCache = async (client: Client): Promise<void> => {
 }
 
 const stats = async (message: Message, _) => {
-  const embedMessage = buildEmbed(message.client['guildData'].get(`${message.guild.id}.stats`))
-  return message.channel.send(embedMessage)
+  const guildStats = message.client['guildData'].get(`${message.guild.id}.stats`)
+  const embedMessage: RichEmbed = buildEmbed(guildStats)
+  const statsMessage: Message | Message[] = await message.channel.send(embedMessage)
```

Comment:
Is it possible to just cast this to Message once and not three times?

Reply:
I'd have to investigate a bit after i'm done working. Not sure if something like (statsMessage as Message): Message | Message[] would work